src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -49,6 +49,8 @@ public static class CPythonEvaluator
                     PythonEngine.Initialize();
                     PythonEngine.BeginAllowThreads();
                 }
+                
+                prev_code = code;
```

Comment:
So I guess check in line 43 was not really working and now it is. Probably not what we want to do, given that we are now getting null as a result if the code didn't change. I'd say we get rid of the check unless @reddyashish says otherwise.

Reply:
We do a similar check in the ironPythonEvaluator upon which this code is based: 
https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/DSIronPython/IronPythonEvaluator.cs#L104

I think it is avoid re-executing and importing libraries unnecessarily, I'm not sure what it means it this context, but it looks like the python std lib is reloaded somehow if we don't make this check?

Reply:
But right now it doesn't seem correct. If you take a look at CPython, it's returning null when the code did not change. For IronPython that is not the case. I don't think the optimization as it is now in CPython is safe in all cases.

Reply:
ahh, I see, let me see if theres a smarter place to put this check.

Reply:
@mmisol I think you are right - I got rid of it, heres the setupPython code:
https://github.com/henon/Python.Included/blob/master/Python.Included/Installer.cs#L37

we're not calling `force` so this code should only end up loading python once.